frontend/js/src/Profile.tsx

```diff
@@ -331,7 +333,7 @@ function UserProfile({ user, csrf_token }: ProfileProps): JSX.Element {
   return (
     <>
       <h3>Contact information</h3>
-      <p>
+      <div className="mb-3">
```

Comment:
I like the idea of a margin here, but AFAICT the class does not currently exist in MeB